inc/cropper/namespace.php

```diff
@@ -996,9 +996,7 @@ function get_img_src_dimensions( string $image_src, array $image_meta ) {
 	} elseif ( isset( $tachyon_args['fit'] ) ) {
 		// Image is uncropped.
 		list( $width, $height ) = explode( ',', $tachyon_args['fit'] );
-		if ( empty( $height ) ) {
-			list( $width, $height ) = wp_constrain_dimensions( $image_meta['width'], $image_meta['height'], $width );
-		}
+		list( $width, $height ) = wp_constrain_dimensions( $image_meta['width'], $image_meta['height'], $width, $height ?? 0 );
```

Comment:
Drive-by comment: why are you redefining $width and $height? Should the previous line be removed?

Reply:
Oh, good catch

Reply:
It shouldn't be removed but it could potentially use different variable names. The variable names are suitably descriptive in both cases though. The first line gives the width and height to fit the image to, the second gives the width and height after being constrained to match the actual image dimensions.

Reply:
Doesn't the second line effectively overwrite the `$width` and `$height` variables defined in the first line, though?

Reply:
The entire function is overwriting those variables as it goes to reach the final valid version of them. Overwriting variables isn't inherently bad 😄  Would recommend watching back @tfrommen's workshop on performance, it has some really interesting points.